.vale/styles/Microsoft/AMPM.yml

```diff
@@ -0,0 +1,9 @@
+extends: existence
```

Comment:
Do we need to _copy_ all the Microsoft rules here, or could we instead reference a specific version of theirs somehow?

If not possible, we should add a README description about how to update/sync these files, please.

Reply:
The copying is a consequence of executing `vale sync`. Without these files, these third party rules cannot be used for linting.

I will add some info to the README.

Reply:
I'm wondering if shouldn't run `vale sync` on CI. If we don't, CI always checks what we have in the repo. If we do, CI runs something different, that might lead to some drift.

I'm for removing the `vale sync` and maybe look for some automation for updates. Although I don't know how often we need to update that.

Reply:
Good point

Reply:
Done